static/src/javascripts/projects/common/modules/commercial/third-party-tags/outbrain.js

```diff
@@ -110,7 +110,7 @@ define([
                 && !config.page.isPreview
                 && this.identityPolicy()
                 && config.page.section !== 'childrens-books-site') {
-                if (detect.adblockInUse()) {
+                if (detect.adblockInUse() || config.page.edition.toLowerCase() === 'int') {
```

Comment:
I'm not sure if it's explicit, looking at this code, _why_ we can load instantly on the international edition. Maybe an inner function like `hasHighRelevanceComponent` might be more obvious? Or at least a note in the test?